stores/user_store.jsx

```diff
@@ -459,6 +459,22 @@ class UserStoreClass extends EventEmitter {
         return this.getStatuses()[id] || UserStatuses.OFFLINE;
     }
 
+    getTimezone(id) {
+        const profile = store.getState().entities.users.profiles[id];
+        if (profile && profile.timezone) {
+            return {
+                ...profile.timezone,
+                useAutomaticTimezone: profile.timezone.useAutomaticTimezone === 'true',
+            };
+        }
+
+        return {
```

Comment:
Do we care whether the timezone is automatic or manual in most places? It might make sense to just have a function that returns the timezone string without the rest of the object.

Reply:
This is useful for the settings where we do need reference to the entire object, I agree that it would be great to only return a timezone string.

That's what the selector will be used for.  Maybe it would be great to distinguish between `settingsTimezone` object and `userTimezone` string in the function name